.pre-commit-config.yaml

```diff
@@ -0,0 +1,22 @@
+exclude: ^docs/
+
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.5.0
+  hooks:
+  - id: check-yaml
+  - id: check-case-conflict
+  - id: check-json
+  - id: check-merge-conflict
+  - id: check-symlinks
+  - id: check-toml
+  - id: end-of-file-fixer
+  - id: requirements-txt-fixer
+  - id: detect-private-key
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  rev: v0.3.5
+  hooks:
+    - id: ruff
+      args: [ --fix ]
+    - id: ruff-format
+    
```

Comment:
please configure isort to sort the inputs in this order

- built in packages
- third party python packages
- django imports
- zango imports
- relative imports